http_request_randomizer/requests/proxy/requestProxy.py

```diff
@@ -99,7 +99,7 @@ def generate_proxied_request(self, url, method="GET", params={}, data={}, header
 
             self.logger.debug("Using proxy: {0}".format(str(self.current_proxy)))
             request = requests.request(method, url, proxies={"http": self.current_proxy},
-                                       headers=headers, data=data, params=params, timeout=req_timeout)
+                                       headers=headers, data=data, params=params, timeout=req_timeout, allow_redirects=False)
```

Comment:
Since proxy providers could use redirects in some cases, I would recommend using the **maxRedirects** property instead - a default value could be provided as method argument (i.e 10)

Let me know what you think